src/models/identity.py

```diff
@@ -1,17 +0,0 @@
-from torch import nn, Tensor
-
-from src.models.base import BaseModel
-
-
-class Identity(BaseModel):
```

Comment:
Why did you remove an Identity module? It should be in the backbones, heads and pooling registries

Reply:
Because we decide not to use Identity in tasks, fro example in classification task we use
`neck_name = self._hparams.task.params.get('neck_name')` for get neck name but we can use
`neck_name = self._hparams.task.params.get('neck_name', 'Identity')`. So why we need this modules?